src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -736,13 +737,20 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
     protected function getContainerLoader(ContainerInterface $container)
     {
         $locator = new FileLocator($this);
-        $resolver = new LoaderResolver(array(
+        $loaders = array(
             new XmlFileLoader($container, $locator),
             new YamlFileLoader($container, $locator),
             new IniFileLoader($container, $locator),
             new PhpFileLoader($container, $locator),
             new ClosureLoader($container),
-        ));
+        );
+
+        // DirectoryLoader only appears in DI 2.7
+        if (class_exists('Symfony\\Component\\DependencyInjection\\Loader\\DirectoryLoader')) {
```

Comment:
Are you sure about that? Most of the time, you will have latest version, but if you do require an old version, it’ll still work, just not that part. Seems like a rather small feature to have such a big version bump.